content/content_test.go

```diff
@@ -136,10 +136,3 @@ func TestContentParseNoInternal(t *testing.T) {
 	_, err := content.ParseRuleContentDir(noInternalPath)
 	assert.EqualError(t, err, fmt.Sprintf("open %s/internal: no such file or directory", noInternalPath))
 }
-
-// TestContentParseNoReason tests failing when no reason file in rule or error key dirs is present
-func TestContentParseNoReason(t *testing.T) {
```

Comment:
I wouldn't remove the test, but instead make it work when reason.md is not present. Can that be done?

Reply:
it can, in the same way we did it for `summary.md`

Reply:
but as the quick fix, prob. it's ok :p

Reply:
The test didn't even work, the tests passed even if the reason.md was missing so I didn't bother with it

Reply:
The test doesn't check against real rules and we have warn logs if they're empty during the real parsing. These are the important messages that were hidden in the recent problem